src/Test/WebTestCase.php

```diff
@@ -72,11 +72,6 @@
      */
     private $excludedDoctrineTables = [];
 
-    /**
-     * @var array
-     */
-    private static $cachedMetadatas = [];
```

Comment:
I don't understand why this change is here, since it was removed in your previous PR: https://github.com/liip/LiipFunctionalTestBundle/pull/398/files#diff-3216cb8554ab63f3b7b8531ebcee5156

Maybe rebasing your PR on the `2.x` could fix this issue?

Reply:
Hm.. I'll check it today

Reply:
Done